### DISCUSSION TITLE
Challenge Hash is always the same?? !! ??

### DISCUSSION THREAD

**Discussion by GeorgePapageorgakis** [Chia-Network/chia-blockchain#6307] [Support]
I have 2 full node farmers in my private network and in the Last Attempted Proof I always see the same Challenge Hash.
Is this normal?

**Comment by moshenskyDV** [comment]
Not always. You see each challenge hash 64 times. Challenge round contains 64 signage_points, and in the end to calculate passing filter chances you use sha256(plotId+challenge_hash+signage_point), so filter variable will be different every time.

**Comment by BrandtH22** [comment]
@GeorgePapageorgakis , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
